### PR TITLE
Update links with master to use main

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Updating master references to main.
The changes here are not much so am doing it as a PR.

### CODE REVIEW

File: eng/common/pipelines/templates/steps/docs-metadata-release.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ parameters:
     default: ''
   - name: PRBranchName
     type: string
-    default: 'master-rdme'
+    default: 'main-rdme'
```
Comment (chidozieononiwu) [inline eng/common/pipelines/templates/steps/docs-metadata-release.yml:29]: @scbedd  Do you know if this affect anything else?

File: .azure-pipelines/apiview.yml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 trigger:
   branches:
     include:
-      - master
       - feature/*
```
Comment (weshaggard) [inline .azure-pipelines/apiview.yml:4]: Please add main to the lists here as well as it is missing.

**Review by weshaggard — APPROVED** [review]
Make sure main is added to all the yml files while removing master but otherwise looks good.

**Review by mitchdenny — APPROVED** [review]
Approved pending response from @scbedd

**Comment by ONFU** [comment]
the nable ad phaaron egyth districk hab kharha am touthenkamoin and lest and sland goring having destroye by omega of ramses hab kharha my reing of thwe omega disturb haved changing by leaything lntendant of my NAuthority and authority and my districk of Nautoharha of my NoX-PacT

**Pull request merged**